src/test/ui/lint/not_found.stderr

```diff
@@ -0,0 +1,20 @@
+warning: unknown lint: `foo_bar`
+  --> $DIR/not_found.rs:11:9
+   |
+14 | #[allow(FOO_BAR)]
+   |         ^^^^^^^
+   |
+   = note: #[warn(unknown_lints)] on by default
+
+warning: unknown lint: `DEAD_CODE`
+  --> $DIR/not_found.rs:12:8
+   |
+16 | #[warn(DEAD_CODE)]
+   |        ^^^^^^^^^ help: lowercase the lint name: `dead_code`
+
+warning: unknown lint: `Warnings`
+  --> $DIR/not_found.rs:13:8
```

Comment:
The line numbers are wrong.

```diff
-warning: unknown lint: `foo_bar`
-  --> $DIR/not_found.rs:11:9
+warning: unknown lint: `FOO_BAR`
+  --> $DIR/not_found.rs:14:9
    |
 14 | #[allow(FOO_BAR)]
    |         ^^^^^^^
    |
    = note: #[warn(unknown_lints)] on by default
 
 warning: unknown lint: `DEAD_CODE`
-  --> $DIR/not_found.rs:12:8
+  --> $DIR/not_found.rs:16:8
    |
 16 | #[warn(DEAD_CODE)]
    |        ^^^^^^^^^ help: lowercase the lint name: `dead_code`
 
 warning: unknown lint: `Warnings`
-  --> $DIR/not_found.rs:13:8
+  --> $DIR/not_found.rs:18:8
    |
 18 | #[deny(Warnings)]
    |        ^^^^^^^^ help: lowercase the lint name: `warnings`
```

Reply:
Yeah, sorry about that. Me, at a bus stop, with slow, intermittent 3G connection trying to fake compiletest (because rustbuild doesn't like connection timeouts). Let's try that again.